Backend/src/main/java/com/onbank/TestEndPoint.java

```diff
@@ -6,8 +6,17 @@
 @RestController
 public class TestEndPoint {
 
-    @GetMapping(path = {"test", "hello"})
+    @GetMapping("hello")
     public String getTest(){
         return "Hello world";
     }
+    @GetMapping("null")
+    public String getNullEx(){
```

Comment:
Czemu ma służyć taki endpoint?

Reply:
Jeżeli chcecie testować Advice to powinniście zrobić testy jednostkowe z wykorzystaniem MockMvc

Reply:
Jednak na razie jako że macie tylko jeden endpoint z hello to nie ma za bardzo co testować, jeżeli Advice działa na tych endpointach to ok, ale nie ma sensu tego wrzucać do repozytorium bo nie spełnia żadnej biznesowej funkcjonalności. Gdy będziecie robić endpointy zwracające już jakieś dane wtedy testy jednostkowe będą niezbędne.

Reply:
Te endpointy służyły tylko po to aby pokazać, że wszystkie rodzaje błędów są obsłużone (te zaimplementowane i te niezaimplementowane).

Reply:
Klient nie będzie zważał na takie demonstracje. Jeżeli chcesz udowodnić że jakiś scenariusz jest obsłużony w kodzie, to należy zrobić to za pomocą testów. W kontrolerze powinny znajdować się wyłącznie metody, z których faktycznie korzysta Frontend

Reply:
Ale cały controller TestEndPoint jest niepotrzebny w aplikacji, gdyż został napisany tylko bo był taki wymóg w tworzeniu szkieletu. Tzn. że już możemy go usunąć z aplikacji?